Unicode/ustring.c

```diff
@@ -1031,6 +1041,18 @@ int u_endswith(const unichar_t *haystack,const unichar_t *needle) {
     return p == ( haystack + haylen - nedlen );
 }
 
+int u_endswithi(const unichar_t *haystackZ,const unichar_t *needleZ) {
+    unichar_t* haystack = u_copytolower(haystackZ);
+    unichar_t* needle   = u_copytolower(needleZ);
```

Comment:
I didn't find where you call this. What is it for?

As written this will lean haystack and needle. And it seems like the remaining lines are exactly the same as `u_endswith()`. Why can't you do:

```
    int ret;
    unichar_t* haystack = u_copytolower(haystackZ);
    unichar_t* needle   = u_copytolower(needleZ);
    ret = u_endswith(haystack, needle);
    free(haystack);
    free(needle);
    return ret;
```
?

Reply:
I do call it, I just forgot to put it in this branch, it ended up in my `git stash`, sorry. Fixed in https://github.com/fontforge/fontforge/pull/3769/commits/2ff0dc1ba00dbac63a1082a9fe6e8b834f9408e6

Reply:
If it ain't in the branch it ain't \[something clever that rhymes with "branch"\]